common/components/common/upload/FileUploadButton.jsx

```diff
@@ -12,7 +13,8 @@ export type FileUploadData = {|
 type Props = {|
   onFileUpload: (FileUploadData) => void,
   buttonText: string,
-  acceptedFileTypes: string
+  acceptedFileTypes: string,
+  faIconClass: string
```

Comment:
I would name this just iconClass, since we could easily swap this out with a glyph library other than FontAwesome.